HTML/mockup-mogo/js/task6.js

```diff
@@ -0,0 +1,27 @@
+import {loadedElements} from './elements.js';
+
+export function displayAccordion() {
+  loadedElements.then((elements) => {
+    elements.optionsPart.addEventListener('click', (e) => {
+      let targetClosestOption = e.target.closest('.option-item__top');
+
+      if (!targetClosestOption) {
+        return false;
+      }
+
+      let targetIndex = targetClosestOption.dataset.index;
+
+      elements.optionTop.forEach((el, index) => {
```

Comment:
form js part optionTop it is more like accordionSectionToggler or something like this